client/wc-api/settings/selectors.js

```diff
@@ -36,7 +37,26 @@ const isGetSettingsRequesting = getResource => group => {
 		return true;
```

Comment:
I'm not getting the notice to appear. `lastReceived` is undefined when I stop the and the value returned is always `true` so no notice gets generated.

Can you reproduce?

Reply:
![settings-save](https://user-images.githubusercontent.com/22080/61553822-29438400-aa10-11e9-9566-bf651da43a11.gif)

Just tested again locally, and all is working for me still.

Reply:
Ok, I must have goofed something up because its working for me now